src/Auth0.php

```diff
@@ -8,6 +8,9 @@
 use Auth0\SDK\Store\SessionStore;
 use Auth0\SDK\Store\StoreInterface;
 use Auth0\SDK\API\Authentication;
+use Auth0\SDK\API\Helpers\State\StateHandler;
```

Comment:
I don't see `StateHandler` being instantiated or referenced here. I do see `SessionStateHandler` and `DummyStateHandler` though

Reply:
It's mentioned in a docblock, which is flagged in my IDE